pkg/queue/metrics.go

```diff
@@ -0,0 +1,68 @@
+package queue
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// largest bucket is 5 seconds
+var durationMsBuckets = []float64{10, 50, 100, 200, 300, 500, 1000, 2000, 3000, 5000}
+
+var (
+	queueMetricLabels = []string{"queue"}
+	taskMetricLabels  = []string{"queue", "type"}
+)
+
+// TaskQueueMetrics provides access to the prometheus metric objects for the task queue
+var TaskQueueMetrics = struct {
+	Labels          []string
+	TaskCounter     *prometheus.CounterVec
+	EnqueueDuration *prometheus.HistogramVec
+}{
+	Labels: queueMetricLabels,
+	TaskCounter: promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "hub",
```

Comment:
this can't be here, we need a function that creates the metrics and probably store them in the queue implementation.

Reply:
fixed